atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -253,7 +252,18 @@ public void compactInternally() {
                         + " IOT tables to compensate for bloat. You can contact Palantir Support if you'd"
                         + " like more information. Underlying error was: {}", tableRef, e.getMessage());
             } catch (TableMappingNotFoundException e) {
-                throw Throwables.propagate(e);
+                throw new RuntimeException(e);
+            }
+        } else {
+            try {
+                log.warn("Tried to clean up table {}, but couldn't, because enterprise features are disabled. "
```

Comment:
`Tried to clean up {} bloat after a sweep operation`: to be consistent with the above. In fact we should be able to consolidate the two error messages with the difference that Enterprise Edition features are not enabled in the later case.